app/models.py

```diff
@@ -1044,6 +1056,7 @@ def formatted_status(self):
                 'technical-failure': 'Technical failure',
                 'sending': NOTIFICATION_STATUS_LETTER_ACCEPTED_PRETTY,
                 'created': NOTIFICATION_STATUS_LETTER_ACCEPTED_PRETTY,
+                'delivered': 'Received'
```

Comment:
Use the defined const? NOTIFICATION_STATUS_LETTER_RECEIVED?

Reply:
Defined const is in lower case, for it to be nicer looking it's capitalised here, and since it's only used here I haven't made it a variable